modules/renter/contractor/contracts.go

```diff
@@ -265,15 +265,13 @@ func (c *Contractor) managedRenew(contract modules.RenterContract, numSectors ui
 	}
 
 	// create contract params
-	c.mu.RLock()
 	params := proto.ContractParams{
 		Host:          host,
 		Filesize:      numSectors * modules.SectorSize,
-		StartHeight:   c.blockHeight,
+		StartHeight:   newStartHeight,
```

Comment:
any particular reason for changing the start height to be different from the current block height? I think that the old method of using the block height was okay. Unless I'm remembering these fields wrong, 'StartHeight' is supposed to indicate the block that was used to form the contract.